bloom-housing/bloom
# feat: update ami chart script

This PR addresses 🤷 


## Description

New script runner endpoint to update a AMI Charts values based on the AMI Chart ID.

## How Can This Be Tested/Reviewed?

Sign in as an admin user to the partners portal.
Pick an AMI Chart to update. Copy the ID.
Using OpenAPI or ThunderClient, call the `/scriptRunner/amiChartUpdateImport` by passing in the id and the correctly formatted values.
Confirm in the partners portal that the AMI Chart values are appearing as expected.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
